lib/shopify_app/configuration.rb

```diff
@@ -70,5 +73,8 @@ def self.configuration=(config)
 
   def self.configure
     yield configuration
+    if !ShopifyApp.configuration.embedded_app? || @disable_same_site_cookie_middleware
```

Comment:
This expresses the same thing, I think

```suggestion
    unless ShopifyApp.configuration.embedded_app?
```

Reply:
(As discussed in person) It seems like this logic could be simpler if the option were phrased positively, i.e. `set_same_site_none`. This would also hide that we're using a middleware from our public interface in case we want to change it later (which we probably will).

We could then write it as:

```
if @set_cookies_same_site_none.nil?
   @set_cookies_same_site_none = ShopifyApp.configuration.embedded_app?
```